packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts

```diff
@@ -572,21 +496,18 @@ async function insertNodesToTestTree(
 async function setNodesNarrow(
 	tree: ISharedTree,
 	numberOfNodes: number,
-	dataType: string,
 	provider: ITestTreeProvider,
 ): Promise<void> {
 	let currPath: UpPath = {
 		parent: undefined,
 		parentField: rootFieldKeySymbol,
 		parentIndex: 0,
 	};
-	const node = testSubtrees.get(dataType);
-	assert(node !== undefined);
 	for (let i = 0; i < numberOfNodes; i++) {
-		tree.runTransaction((forest, editor) => {
-			const field = editor.sequenceField(currPath, localFieldKey);
-			field.insert(0, singleTextCursor(node));
-			return TransactionResult.Apply;
+		runSynchronous(tree, () => {
+			const field = tree.editor.sequenceField(currPath, localFieldKey);
+			field.insert(0, singleTextCursor(testTreeNode));
+			return TransactionResult.Commit;
 		});
```

Comment:
With the new branch API, we have mostly separated the concept of editing from transactions. You only need to use runSynchronous if you need to be able to roll back just that batch of changes, or you need to control atomicity of a group of changes. I think you can simplify this to:

```suggestion
		const field = tree.editor.sequenceField(currPath, localFieldKey);
		field.insert(0, singleTextCursor(testTreeNode));
```

Reply:
Similar simplifications should be possible elsewhere

Reply:
removed unnecessary `runSynchronous` calls